types/index.d.ts

```diff
@@ -2210,7 +2210,7 @@ export declare namespace Knex {
     notNullable(): ColumnBuilder;
     nullable(): ColumnBuilder;
     comment(value: string): ColumnBuilder;
-    alter(): ColumnBuilder;
+    alter(options: Readonly<{alterNullable?: boolean, alterType?: boolean}>): ColumnBuilder;
```

Comment:
this is breaking for TS users. the parameter was not there, now its required.

Reply:
Yeah, sorry, just fixed and merged here : https://github.com/knex/knex/pull/4996

Reply:
will publish hotfix shortly